server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -191,7 +141,7 @@ public TSOpenSessionResp openSession(TSOpenSessionReq req) throws TException {
         TSProtocolVersion.IOTDB_SERVICE_PROTOCOL_V1);
     resp.setSessionHandle(
```

Comment:
Is the TS_SessionHandle necessary in the open session response?

Reply:
It seems to be unnecessary now, because TS_SessionHandle only has one TSHandleIdentifier attribute and the guid, secret field in TSHandleIdentifier are useless now.
Maybe, it will be useful when we want to make use of the guid and secret field in TSHandleIdentifier.